src/amplitude-client.js

```diff
@@ -817,7 +817,7 @@ var _convertProxyObjectToRealObject = function _convertProxyObjectToRealObject(i
 AmplitudeClient.prototype.identify = function(identify_obj, opt_callback) {
   if (!this._apiKeySet('identify()')) {
     if (type(opt_callback) === 'function') {
-      opt_callback(0, 'No request sent');
+      opt_callback(0, 'No request sent', 'API key is not set');
```

Comment:
I'd make it an object with a single key, in case we decide to extend the information even more in the future.